src/event.rs

```diff
@@ -748,22 +748,30 @@ pub enum MouseButton {
     Other(u8),
 }
 
-/// Describes a difference in the mouse scroll wheel state.
+/// A measured distance as the user scrolls.
+///
+/// Many modern compositors will express touchpad drags and tablet surface touches in units of
```

Comment:
could we discard tablets information from here? winit doesn't handle those. I'm not sure about the word compositors here, I'd call it systems, or something like that, since some systems like X11 doesn't have. I'd also remove 'modern' part.

In general, I'd just wrote what devices could produce those events, but I feel like it should be above enum variants, not the enum itself, so basically like it was before, so you'll have a cause of event attached to event.

Reply:
I'm not sure I understand some of your suggestions. Since nothing in my PR behaviorally changes how these enums should be used, I'll just revert the doc changes.